ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/state/CrosslinkCommittee.java

```diff
@@ -1,46 +0,0 @@
-/*
- * Copyright 2019 ConsenSys AG.
- *
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
- * the License. You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
- * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
- * specific language governing permissions and limitations under the License.
```

Comment:
Can you add this back? This is a client-specific data structure that we use in some places in the Validator Coordinator. It's not clear that this is not a spec data structure so we should get rid of it or make that clear in the future, but for this spec upgrade, let's keep it.